disnake/guild.py

```diff
@@ -177,7 +177,7 @@ class Guild(Hashable):
         The channel that denotes the AFK channel. ``None`` if it doesn't exist.
     id: :class:`int`
         The guild's ID.
-    owner_id: :class:`int`
+    owner_id: :class:`Optional[int]`
         The guild owner's ID. Use :attr:`Guild.owner` if you need a :class:`Member` object instead.
```

Comment:
A small note explaining why/when this can be None could be helpful:
```suggestion
        The guild owner's ID. Use :attr:`Guild.owner` if you need a :class:`Member` object instead.
        This may be ``None`` if the guild is :attr:`~Guild.unavailable`.
```